src/Plugin/Field/FieldWidget/AutomatedListingConfigurationWidget.php

```diff
@@ -656,7 +744,7 @@ protected function buildFilterOperatorSelect($default_value = 'AND', $descriptio
    *   The reference fields.
    */
   protected function getEntityReferenceFields() {
-    $reference_fields = $this->indexHelper->getIndexEntityReferenceFields($this->index, ['nid']);
+    $reference_fields = $this->indexHelper->getIndexEntityReferenceFields($this->index, ['nid', 'uid']);
```

Comment:
hi @asarfara,

Excluding `uid` this way will make it impossible to create a listing of content items created by a specific user. The module already provides a hook to do so:
```php
/**
 * Implements hook_tide_automated_listing_entity_reference_fields_exclude().
 */
function mymodule_automated_listing_entity_reference_fields_exclude(IndexInterface $index, array $reference_fields = [], FieldItemListInterface $items = NULL, $delta = NULL) {
  $excludes = [];
  if ($index->id() == 'node') {
    $excludes = ['field_node_site', 'field_primary_site', 'uid'];
  }
  return $excludes;
}
```

Alternatively, you can introduce a widget setting to exclude some filter from the Form UI.

Reply:
Yep i like the idea of excluding filters through widget settings. I will update that.